admin/email-admin.php

```diff
@@ -41,39 +41,46 @@ function insert_instant_send_js() {
 	if ( $post->post_type !== 'ucf-email' ) return;
 ?>
 	<script>
+	(function($) {
 		$post_id = <?php echo $post->ID; ?>;
 
 		var data = {
 			post_id: $post_id,
 			action: 'instant-send'
 		};
+		var $sendBtn = $('#instant-send');
+		var $spinner = $('<img src="<?php echo admin_url( '/images/wpspin_light.gif' ); ?>" alt="Processing..." style="margin-left: 6px; display: inline-block; vertical-align: sub;">');
 
 		var onPostSuccess = function(response) {
+			$spinner.remove();
+
 			if ( response.success === true ) {
-				var $markup = jQuery(
+				var $markup = $(
 					'<div class="updated notice notice-success is-dismissible">' +
-						'<p>Preview of email sent.</p>' +
+						'<p>Email test sent.</p>' +
 					'</div>'
 				);
 			} else {
-				var $markup = jQuery(
+				var $markup = $(
 					'<div class="notice notice-error is-dismissible">' +
-						'<p>There was a problem sending the preview.</p>' +
+						'<p>There was a problem sending the email test.</p>' +
 					'</div>'
 				);
 			}
 
 			$markup.insertAfter('.wp-header-end');
 		};
 
-		jQuery('#instant-send').on('click', function() {
-			jQuery.post(
+		$sendBtn.on('click', function() {
+			$sendBtn.append($spinner);
+			$.post(
```

Comment:
You could add $spinner.remove() to .always() callback to ensure the spinner is removed if the ajax call fails.
https://api.jquery.com/jQuery.post/